### PR TITLE
Update links in clocker.bom

### PR DESCRIPTION
**Pull request by m4rkmckenna**

### CODE REVIEW

File: clocker.bom [hunk 1]
```diff
@@ -1,12 +1,12 @@
 brooklyn.catalog:
   brooklyn.libraries:
     - "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=io.brooklyn.etcd&a=brooklyn-etcd&v=2.3.0-SNAPSHOT"
-#    TODO: Replace below URLs with link to release JARs
-    - "http://localhost:8000/io/brooklyn/clocker/common/2.0.0-SNAPSHOT/common-2.0.0-SNAPSHOT.jar"
-    - "http://localhost:8000/io/brooklyn/clocker/swarm/2.0.0-SNAPSHOT/swarm-2.0.0-SNAPSHOT.jar"
-    - "http://localhost:8000/io/brooklyn/clocker/kubernetes/2.0.0-SNAPSHOT/kubernetes-2.0.0-SNAPSHOT.jar"
+    - "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=io.brooklyn.clocker&a=common&v=2.0.0"
```
Comment (grkvlt) [inline clocker.bom]: Change this to `https://oss.sonatype.org/service/local/repositories/releases/content/io/brooklyn/clocker/common/2.0.0/common-2.0.0.jar`

File: clocker.bom [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+    - "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=io.brooklyn.clocker&a=swarm&v=2.0.0"
+    - "https://oss.sonatype.org/service/local/artifact/maven/redirect?r=releases&g=io.brooklyn.clocker&a=kubernetes&v=2.0.0"
```
Comment (grkvlt) [inline clocker.bom]: Similar changes for the other two Jar files

Comment (m4rkmckenna) [inline clocker.bom]: Whats wrong with the redirect?

Comment (grkvlt) [inline clocker.bom]: It looks awful...?

Comment (grkvlt) [inline clocker.bom]: Also, copy/paste into a shell will break, unless you remember to escape the `&` - it's just nicer to have the direct URL...

Comment (m4rkmckenna) [inline clocker.bom]: What do you suggest for `brooklyn-etcd` as its a snapshot and the absolute url changes every night?

**Comment by m4rkmckenna** [comment]
@grkvlt Ive updated to absolute URLs where possible

**Pull request merged**